Python/Tests/Core/BuildTasksTests.cs

```diff
@@ -46,7 +46,7 @@ public class BuildTasksTests {
             AssertListener.Initialize();
         }
 
-        //[TestMethod, Priority(3)]
+        //[TestMethod, UnitTestPriority(TestExtensions.P3_UNIT_TEST)]
```

Comment:
You should either undo these or make them use the correct constant so it compiles when we uncomment

Reply:
Done